otel-infrastructure-collector/values.yaml

```diff
@@ -6,7 +6,7 @@ global:
   logs:
     endpoint: "<logs-endpoint-here>"
   defaultApplicationName: "default"
-  defaultSubsystemName: "nodes"
```

Comment:
What is the motivation behind?

Reply:
the logs of the events-exporter dont include any of the subsystem attributes, so its taking the default subsystem, which is 'nodes' , and its weird to have logs of events under nodes. 
actually metrics is not good since its events logs and not metrics, 
but maybe 'events' or something

Reply:
kube-events maybe.